noxfiles/constants_nox.py

```diff
@@ -91,6 +92,16 @@
     "--wait",
     COMPOSE_SERVICE_NAME,
 )
+START_APP_REMOTE_DEBUG = (
```

Comment:
unlike the other `START_*` arg constants, we don't have a `--wait` flag here since we don't want to detach in this case. but we don't have any other arg constants that are calling a `docker compose up` without detaching, so i wasn't totally sure what to choose on the naming convention